c7n/resources/s3control.py

```diff
@@ -95,3 +97,88 @@ class resource_type(TypeInfo):
         permission_prefix = 's3'
 
     source_mapping = {'describe': MultiRegionAccessPointDescribe}
+
+
+class StorageLensDescribe(DescribeSource):
+    def get_query_params(self, query_params):
+        query_params = query_params or {}
+        query_params['AccountId'] = self.manager.config.account_id
+        return query_params
+
+    def augment(self, resources):
+        client = local_session(self.manager.session_factory).client('s3control')
+        results = []
+        for r in resources:
+            storage_lens_configuration = self.manager.retry( \
+                client.get_storage_lens_configuration,
+                AccountId=self.manager.config.account_id,
+                ConfigId=r['Id']) \
+                .get('StorageLensConfiguration')
+            results.append(storage_lens_configuration)
+        return universal_augment(
```

Comment:
We could just change it to 
```return universal_augment(self.manager, super().augment(results))```

Reply:
@PratMis , sounds good. I simplified the code.